build-tools/xaprepare/xaprepare/ToolRunners/MSBuildRunner.cs

```diff
@@ -34,6 +34,7 @@ public async Task<bool> Run (string projectPath, string logTag, List<string>? ar
 			AddArguments (runner, StandardArguments);
 			if (!String.IsNullOrEmpty (binlogName)) {
 				string logPath = Utilities.GetRelativePath (workingDirectory!, Path.Combine (Configurables.Paths.BuildBinDir, $"msbuild-{Context.BuildTimeStamp}-{binlogName}.binlog"));
+				runner.AddQuotedArgument ("/restore");
```

Comment:
`/restore` should be added to `StandardArguments` in the constructor. It should be harmless even if it's repeated.